fbchat/client.py

```diff
@@ -748,11 +748,12 @@ def fetchThreadMessages(self, thread_id=None, limit=20, before=None):
 
         return list(reversed([graphql_to_message(message) for message in j['message_thread']['messages']['nodes']]))
 
-    def fetchThreadList(self, offset=0, limit=20):
+    def fetchThreadList(self, offset=0, limit=20, type='inbox'):
         """Get thread list of your facebook account
 
         :param offset: The offset, from where in the list to recieve threads from
         :param limit: Max. number of threads to retrieve. Capped at 20
+        :param type: (optional) "inbox", "pending", "archived"
```

Comment:
Following the pratice in `fbchat`, perhaps we should use an `Enum` instead of string values? This would allow it to be reused across different functions, and it looks neat and clean ;)

Reply:
OK, Thanks!